libs/com.ericsson.otp.jinterface/src/com/ericsson/otp/erlang/AbstractConnection.java

```diff
@@ -425,29 +425,12 @@ protected void sendUnlink(final OtpErlangPid from, final OtpErlangPid dest,
         header.write1(passThrough);
         header.write1(version);
 
-        if ((peer.flags & AbstractNode.dFlagUnlinkId) != 0) {
-            // header
-            header.write_tuple_head(4);
-            header.write_long(unlinkIdTag);
-            header.write_long(unlink_id);
-            header.write_any(from);
-            header.write_any(dest);
-        }
-        else {
-            /*
-             * A node that isn't capable of talking the new link protocol.
-             *
-             * Send an old unlink op, and send ourselves an unlink-ack. We may
-             * end up in an inconsistent state as we could before the new link
-             * protocol was introduced...
-             */
-            // header
-            header.write_tuple_head(3);
-            header.write_long(unlinkTag);
-            header.write_any(from);
-            header.write_any(dest);
-            deliver(new OtpMsg(unlinkIdAckTag, dest, from, unlink_id));
-        }
+        // header
```

Comment:
I can't review these changes. Please describe in the PR how the uplift was done, so I know what needs to be reviewed and what not.

Reply:
The commit with the jinterface-update has info where its copied from, and the plan is to merge the commits separately. I will be helpful for future updates.
The other commits are mainly the same steps we did in previous lifts; a search and replace.